solana/tests/utils/helpers.ts

```diff
@@ -166,6 +167,15 @@ export class TestsHelper {
     return sendAndConfirm(this.connection, ixs, payer, ...signers);
   }
 
+  async getTransaction(
+    signature: TransactionSignature,
+  ): Promise<VersionedTransactionResponse | null> {
+    return this.connection.getTransaction(signature, {
+      commitment: 'confirmed',
```

Comment:
Hardcoded commitment levels are almost always a no-no. Why not rely on the default commitment level of the connection?

Reply:
I use the helper's finality